creator-node/src/fileManager.js

```diff
@@ -350,7 +350,7 @@ const trackFileUpload = multer({
       cb(null, true)
     } else {
       req.fileFilterError = `File type not accepted. Must be one of [${ALLOWED_UPLOAD_FILE_EXTENSIONS}] with mime type matching ${AUDIO_MIME_TYPE_REGEX}, got file ${fileExtension} with mime ${file.mimetype}`
-      cb(null, false)
+      cb(new Error(req.fileFilterError), false)
```

Comment:
This needs another look, whenever you have the time @SidSethi, Thanks

Reply:
from the multer docs it looks like if throwing an error you don't need to pass second argument
otherwise i think this is fine but it touches critical path so need to carefully test